### PR TITLE
tiflash: Clean deprecated configs; align style of flash.proxy

### PR DESCRIPTION
**Pull request by JaySon-Huang**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

* The config item in `flash.flash_cluster` are deprecated since v6.0 https://github.com/pingcap/tidb/issues/29924
* Align the style of `flash.proxy` with other config items

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [x] v7.6 (TiDB 7.6 versions)
- [x] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/16111
- Other reference link(s): https://github.com/pingcap/tidb/issues/29924

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tiflash/tiflash-configuration.md [hunk 1]
```diff
@@ -155,30 +155,29 @@ delta_index_cache_size = 0
...
-    data-dir = The data storage path of proxy.
-    config = The configuration file path of proxy.
-    log-file = The log path of proxy.
-    log-level = The log level of proxy. "info" is used by default.
-    status-addr = The listening address from which the proxy pulls metrics | status information. If it is left empty, 127.0.0.1:20292 is used by default.
-    advertise-status-addr = The external access address of status-addr. If it is left empty, "status-addr" is used by default.
+    ## The listening address of proxy. If it is left empty, 127.0.0.1:20170 is used by default.
+    addr = "127.0.0.1:20170"
+    ## The external access address of addr. If it is left empty, "addr" is used by default.
+    advertise-addr = ""
```
Comment (CalvinNeo) [inline tiflash/tiflash-configuration.md:162]: If use 127.0.0.1 as advertise addr, peers on other nodes can't connect to the current node, should we emphasis that?

File: tiflash/tiflash-configuration.md [hunk 2]
```diff
@@ -155,30 +155,29 @@ delta_index_cache_size = 0
...
-    advertise-addr = The external access address of addr. If it is left empty, "addr" is used by default.
-    data-dir = The data storage path of proxy.
-    config = The configuration file path of proxy.
-    log-file = The log path of proxy.
-    log-level = The log level of proxy. "info" is used by default.
-    status-addr = The listening address from which the proxy pulls metrics | status information. If it is left empty, 127.0.0.1:20292 is used by default.
-    advertise-status-addr = The external access address of status-addr. If it is left empty, "status-addr" is used by default.
+    ## The listening address of proxy. If it is left empty, 127.0.0.1:20170 is used by default.
+    addr = "127.0.0.1:20170"
+    ## The external access address of addr. If it is left empty, "addr" is used by default.
```
Comment (CalvinNeo) [inline tiflash/tiflash-configuration.md:161]:
```suggestion
    ## The external access address of addr. If it is left empty, "addr" is used by default.
    ## Should guarantee that other nodes can access though `advertise-addr`.
```

File: tiflash/tiflash-configuration.md [hunk 3]
```diff
@@ -155,30 +155,29 @@ delta_index_cache_size = 0
...
+    advertise-addr = ""
+    ## The listening address from which the proxy pulls metrics | status information. If it is left empty, 127.0.0.1:20292 is used by default.
+    status-addr = "127.0.0.1:20292"
+    ## The external access address of status-addr. If it is left empty, "status-addr" is used by default.
```
Comment (CalvinNeo) [inline tiflash/tiflash-configuration.md:165]:
```suggestion
    ## The external access address of status-addr. If it is left empty, "status-addr" is used by default.
    ## Should guarantee that other nodes can access though `advertise-status-addr`.
```

**Review by CalvinNeo — APPROVED** [review]
LGTM, with some comments

File: tiflash/tiflash-configuration.md (hunk 1 above)
Comment (ran-huang) [reply tiflash/tiflash-configuration.md:162]: @JaySon-Huang PTAL

File: tiflash/tiflash-configuration.md [hunk 4]
```diff
@@ -155,30 +155,31 @@ delta_index_cache_size = 0
...
-    data-dir = The data storage path of proxy.
-    config = The configuration file path of proxy.
-    log-file = The log path of proxy.
-    log-level = The log level of proxy. "info" is used by default.
-    status-addr = The listening address from which the proxy pulls metrics | status information. If it is left empty, 127.0.0.1:20292 is used by default.
-    advertise-status-addr = The external access address of status-addr. If it is left empty, "status-addr" is used by default.
+    ## The listening address of proxy. If it is left empty, 127.0.0.1:20170 is used by default.
+    addr = "127.0.0.1:20170"
+    ## The external access address of addr. If it is left empty, "addr" is used by default.
+    ## Should guarantee that other nodes can access through `advertise-addr`.
```
Comment (JaySon-Huang) [inline tiflash/tiflash-configuration.md:162]:
```suggestion
    ## Should guarantee that other nodes can access through `advertise-addr` when you deploy the cluster on multiple nodes.
```

File: tiflash/tiflash-configuration.md [hunk 5]
```diff
@@ -155,30 +155,31 @@ delta_index_cache_size = 0
...
+    advertise-addr = ""
+    ## The listening address from which the proxy pulls metrics | status information. If it is left empty, 127.0.0.1:20292 is used by default.
+    status-addr = "127.0.0.1:20292"
+    ## The external access address of status-addr. If it is left empty, "status-addr" is used by default.
+    ## Should guarantee that other nodes can access though `advertise-status-addr`.
```
Comment (JaySon-Huang) [inline tiflash/tiflash-configuration.md:167]:
```suggestion
    ## Should guarantee that other nodes can access though `advertise-status-addr` when you deploy the cluster on multiple nodes.
```

File: tiflash/tiflash-configuration.md [hunk 6]
```diff
@@ -155,30 +155,31 @@ delta_index_cache_size = 0
...
-    log-file = The log path of proxy.
-    log-level = The log level of proxy. "info" is used by default.
-    status-addr = The listening address from which the proxy pulls metrics | status information. If it is left empty, 127.0.0.1:20292 is used by default.
-    advertise-status-addr = The external access address of status-addr. If it is left empty, "status-addr" is used by default.
+    ## The listening address of proxy. If it is left empty, 127.0.0.1:20170 is used by default.
+    addr = "127.0.0.1:20170"
+    ## The external access address of addr. If it is left empty, "addr" is used by default.
+    ## Should guarantee that other nodes can access through `advertise-addr` when you deploy the cluster on multiple nodes.
+    advertise-addr = ""
+    ## The listening address from which the proxy pulls metrics | status information. If it is left empty, 127.0.0.1:20292 is used by default.
```
Comment (ran-huang) [inline tiflash/tiflash-configuration.md:164]:
```suggestion
    ## The listening address from which the proxy pulls metrics or status information. If it is left empty, 127.0.0.1:20292 is used by default.
```

File: tiflash/tiflash-configuration.md [hunk 7]
```diff
@@ -155,30 +155,31 @@ delta_index_cache_size = 0
...
+    status-addr = "127.0.0.1:20292"
+    ## The external access address of status-addr. If it is left empty, "status-addr" is used by default.
```
Comment (ran-huang) [inline tiflash/tiflash-configuration.md:166]:
```suggestion
    ## The external access address of status-addr. If it is left empty, the value of "status-addr" is used by default.
```

File: tiflash/tiflash-configuration.md [hunk 8]
```diff
@@ -155,30 +155,31 @@ delta_index_cache_size = 0
...
+    ## Should guarantee that other nodes can access though `advertise-status-addr` when you deploy the cluster on multiple nodes.
```
Comment (ran-huang) [inline tiflash/tiflash-configuration.md:167]:
```suggestion
    ## Should guarantee that other nodes can access through `advertise-status-addr` when you deploy the cluster on multiple nodes.
```

**Pull request merged**